firebase-common/src/main/java/com/google/firebase/heartbeatinfo/HeartBeatInfo.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.heartbeatinfo;
+
+import org.jetbrains.annotations.NotNull;
+
+/** Component that publishes heartbeat information */
+public interface HeartBeatInfo {
+  enum HeartBeat {
+    NO_HEART_BEAT,
+    SDK_HEART_BEAT,
+    GLOBAL_HEART_BEAT,
+    COMBINED_HEART_BEAT,
```

Comment:
I'd recommend you don't repeat "heartbeat" in the names of the enum values and go with e.g.:
`NONE, SDK, GLOBAL, COMBINED`